source/features/comment-on-draft-pr-indicator.tsx

```diff
@@ -17,7 +22,10 @@ function addDraftBanner(newCommentField: HTMLElement): void {
 }
 
 function init(signal: AbortSignal): void {
-	observe('.CommentBox file-attachment', addDraftBanner, {signal});
+	observe(`.CommentBox file-attachment:is(
+		[input="fc-new_comment_field"],
+		[input^="fc-new_inline_comment_discussion"]
+	)`, addDraftBanner, {signal});
```

Comment:
Does this selector need anything other than just the two input attribute selector? They seem pretty specific (i.e. drop commentbox file-attachment)

Reply:
Good question, I instinctively chose to append.